orchagent/intfsorch.cpp

```diff
@@ -775,10 +775,6 @@ void IntfsOrch::doTask(Consumer &consumer)
                 }
                 adminStateChanged = true;
             }
-            else if (field == "nat_zone")
```

Comment:
Not sure I understand what is the issue/fix here. Please explain in PR description.

Reply:
duplicate else if (field == "nat_zone") （line731 and line778）